src/views/listview/list-view.html

```diff
@@ -13,12 +13,12 @@
          dnd-moved="$ctrl.dragMoved()"
          dnd-dragend="$ctrl.dragEnd()"
          ng-class="{'drag-original': $ctrl.isDragOriginal(item), 'pf-selectable': $ctrl.selectItems, 'active': $ctrl.isSelected(item), 'disabled': $ctrl.checkDisabled(item), 'list-view-pf-expand-active': item.isExpanded}">
-      <div class="list-group-item-header">
+      <div class="list-group-item-header" ng-class="{'list-group-item-not-selectable' : !$ctrl.config.selectItems && (!$ctrl.config.useExpandingRows || $ctrl.config.compoundExpansion)}">
         <div class="list-view-pf-dnd-drag-items" ng-if="$ctrl.config.dragEnabled === true">
           <div pf-transclude="parent" class="list-view-pf-main-info"></div>
         </div>
         <div ng-class="{'list-view-pf-dnd-original-items': $ctrl.config.dragEnabled === true}">
-          <div class="list-view-pf-expand" ng-if="$ctrl.config.useExpandingRows">
+          <div class="list-view-pf-expand" ng-if="$ctrl.config.useExpandingRows && !$ctrl.config.compoundExpansion">
```

Comment:
At first glance, it appears that row and compound expansion are mutually exclusive? I spoke with Cat, Serena, etc. and we don't want to prevent both from being used at the same time. In fact, there is an openshift.io design that will use both.

Perhaps this property can be renamed something like useCompoundExpansionOnly, just to make it clear?